modules/client-configuration/pages/clients-opensuse.adoc

```diff
@@ -120,3 +120,11 @@ mgr-create-bootstrap-repo --with-custom-channels
 ----
 
 For more information on registering your clients, see xref:client-configuration:registration-overview.adoc[].
+
+
```

Comment:
This is NOT a problem on openSUSE, as any version that had this problem is EoL since years. This can go away.